dotnet3.5/Polly/SynthesizeSpeechExample/SynthesizeSpeech/SynthesizeSpeech.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache - 2.0
```

Comment:
"Apache - 2.0" -> "Apache-2.0" (see https://spdx.org/licenses/)

Reply:
Don't kn ow how the extra space was added but I've fixed it.